src/Sushi.php

```diff
@@ -88,8 +88,24 @@ public function migrate()
             }
 
             foreach ($firstRow as $column => $value) {
-                $type = is_numeric($value) ? 'integer' : 'string';
-
+                 
+                switch (true) {
+                    case is_numeric($value):
+                        $type = 'integer';
+                        break;
+                    case is_array($value):
+                        $type = 'longtext';
+                        break;
+                    case is_float($value):
+                        $type = 'float';
+                        break;
+                    case is_string($value):
+                        $type = 'string';
+                        break;
+                    default:
```

Comment:
I think that adding support for date times would be nice.

```suggestion
                    case is_object($value) && $value instanceof \DateTime:
                        $type = 'timestamp';
                        break;
                    default:
```